.github/workflows/build_publish.yaml

```diff
@@ -0,0 +1,77 @@
+name: Build and Publish bahmni-metabase Image
+on:
+  push:
+    branches: 
+      - main
+      - BAH-2665
```

Comment:
PR looks good. Please remove branch before merge.

Reply:
Sure , We will remove it. Thank you .